test/integration/maincluster_test.go

```diff
@@ -1365,28 +1297,10 @@ worker:
 					if reflect.DeepEqual(expected, p.Experimental) {
 						t.Errorf("experimental settings shouldn't be inherited to a node pool but it did : toplevel=%v nodepool=%v", expected, p.Experimental)
 					}
-
 				},
 			},
 			assertCluster: []ClusterTester{
 				hasDefaultCluster,
-				func(c root.Cluster, t *testing.T) {
-					cp := c.ControlPlane()
-					controllerUserdataS3Part := cp.UserDataController.Parts[model.USERDATA_S3].Asset.Content
-					if !strings.Contains(controllerUserdataS3Part, `--feature-gates=PodPriority=true`) {
-						t.Error("missing controller feature gate: PodPriority=true")
-					}
-
-					if !strings.Contains(controllerUserdataS3Part, `scheduling.k8s.io/v1alpha1=true`) {
-						t.Error("missing controller runtime config: scheduling.k8s.io/v1alpha1=true")
-					}
-
-					re, _ := regexp.Compile("--admission-control=[a-zA-z,]*,Priority")
-					if len(re.FindString(controllerUserdataS3Part)) == 0 {
-						t.Error("missing controller --admission-control config: Priority")
-					}
-
-				},
```

Comment:
Thanks for the follow-up commits!
Btw, would you mind confirming if you intentionally remove some existing test-cases?

Reply:
@amdonov Hi! Can I just revert these removals of test cases, and move forward to merge this?